parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
@adalbertgerald @zuyu One convenient way to print a specific test's output is, under `/build` directory, use command:
```
ctest -VV -R <regular_expression_for_test_name>
```
`-R` stands for regular expression: all tests with names that partially match `<regular_expression_for_test_name>` will be tested. The `-VV` flag is to print the full test output.

For example, in the above case:
```
ctest -VV -R quickstep_parser_tests_ParserTest_create
```

Also for example, to test all `ParseTest`'s (create, insert, etc.):
```
ctest -VV -R ParserTest
```